pkg/build/builder/common.go

```diff
@@ -81,3 +84,47 @@ func updateBuildRevision(c client.BuildInterface, build *api.Build, sourceInfo *
 		glog.Warningf("An error occurred saving build revision: %v", err)
 	}
 }
+
+// runScriptInContainer runs a Bash script in an ephemeral Docker container
+// started off imageID, and returns an error if the script cannot be run or
+// returns a non-zero exit code.
+func runScriptInContainer(client DockerClient, script, imageID string) error {
+	glog.Infof("Running post commit hook with image %s ...", imageID)
```

Comment:
do we want to always print this?  should be V(2) I think. Also I would be more interested in what is actually being run